src/sections/Learn/LearnPage-Sections/learn.js

```diff
@@ -166,33 +184,38 @@ const LearnSection = () => {
       </div>
       <div>
         <h1 className="learn-heading"><span>Meshery - </span> Learn how to manage your service mesh</h1>
+        <Container>
+          <Row Hcenter className="learn-buttons-section labs">
+            <Col xs={12} lg={6}>
+              <Link to="/learn/service-mesh-labs" state={{selectedIndex: 0}}>
+                <Button secondary title="Getting started with any service mesh"/>
```

Comment:
@leecalcote links for these buttons?

Reply:
Links are added in the `Link` to pass the prop

Reply:
I have seen that, but I feel it should be changed.

Reply:
I see what you mean, @Nikhil-Ladha. You're suggesting that the two buttons use the same hyperlink and don't deep link to one of the two (green and yellow) tabs representing the two different categories of labs.

Yes, if we can deep link to have the respective tab active, that would be an improvement.

Reply:
No doubt that this is what the `state={{selectedIndex: x }}` property is about in the Link now., which in my tests is working.

Reply:
I take it back. I'm only seeing the NSM lab when clicking on the "getting started" button.